javascript/javascript1/week4/voice.js

```diff
@@ -0,0 +1,123 @@
+function getReply(command) {
+  let result;
+  if (command.includes("hello my name is")) {
+    const userName = saveName(command);
+    result = `nice to meet you ${userName}`;
+  } else if (command.includes(str2)) {
+    if (!str1) {
+      result = `you have not tell your name yet`;
+    }
+    const yourName = saveName(str1);
+    result = `Your name is ${yourName}`;
+  } else if (command.includes("add")) {
+    const activity = addActivity(command);
+    result = `${activity} added to your todo`;
+  } else if (command.includes("remove")) {
+    result = removeActivity(str5);
+  } else if (command.includes("what is on my")) {
+    const length = `You have ${listArr.length} todos - ${listArr.join(" , ")}`;
+    result = length;
```

Comment:
You can skip creating variables if they are to be used a single time, and directly 
```
return `You have ${listArr.length} todos - ${listArr.join(" , ")}`
```

Reply:
implemented